### ISSUE TITLE
Keyboard navigation on embed block hide change block type button

### ISSUE THREAD

**Issue by truchot** [WordPress/gutenberg#15157]
**Describe the bug**
When navigate with keyboard to vimeo embed block, block mover control comes on top of toolbar control. Arrow-down icon hides change block type button.

**To reproduce**
1. Add paragraph block
2. Add vimeo embed block
3. Set the embed block to "Wide Width"
4. Position cursor in paragraph block
5. Navigate with tab key to vimeo embed block

**Expected behavior**
I wonder if block mover control should be on the left side instead of the top in order to prevent change block type button hidden ?

**Screenshots**
<img width="738" alt="Capture d’écran 2019-04-24 à 20 10 00" src="https://user-images.githubusercontent.com/36343370/56683201-5ad8d780-66cd-11e9-826c-de9587e99a80.png">
<img width="719" alt="Capture d’écran 2019-04-24 à 20 10 18" src="https://user-images.githubusercontent.com/36343370/56683202-5ad8d780-66cd-11e9-9646-1f879b07118d.png">

**Desktop (please complete the following information):**
 - OS: MacOS Mojave
 - Browser : Google Chrome
 - Version 73

**Comment by afercia** [comment]
Note: to reproduce, set the embed block to "Wide Width".

**Comment by truchot** [comment]
I edit steps to reproduce :) 
Thanks !

**Comment by glendaviesnz** [comment]
I wasn't able to replicate this in WP 6.2 so closing.
